certified-connectors/PowerBI Robots/apiDefinition.swagger.json

```diff
@@ -0,0 +1,184 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "PowerBI Robots",
+        "description": "PowerBI Robots connector to execute the internal API across the multiple Power Platform products",
+        "version": "1.0",
+        "contact": {
```

Comment:
Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

Reply:
commited on Identation, response and missing properties corrections